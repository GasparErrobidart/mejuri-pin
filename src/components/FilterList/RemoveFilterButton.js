import styled              from 'styled-components'

const RemoveFilterButton = styled.div`
  color: ${({theme})=> theme.colors.darkMiddleGray};
  margin-left: auto;
  cursor:pointer;
`

export default RemoveFilterButton
