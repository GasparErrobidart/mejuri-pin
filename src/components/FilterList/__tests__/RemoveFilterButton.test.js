import '@testing-library/jest-dom'
import 'jest-styled-components'

import React                 from 'react'
import {ThemeProvider}       from 'styled-components'
import { cleanup }           from '@testing-library/react'
import renderer              from 'react-test-renderer'

import theme                 from '../../GlobalStyle/themes/mejuri-pin'

import RemoveFilterButton   from '../RemoveFilterButton'


afterEach(cleanup)

test('Renders "RemoveFilterButton" component', () => {
  const testText        = "RemoveFilterButton Test Text"
  const component       = renderer.create(
    <ThemeProvider theme={theme}>
      <RemoveFilterButton>{testText}</RemoveFilterButton>
    </ThemeProvider>);
  const tree            = component.toJSON()
  expect(tree).toMatchSnapshot();
});
