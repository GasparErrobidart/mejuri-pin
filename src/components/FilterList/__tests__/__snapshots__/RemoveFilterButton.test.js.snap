// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "RemoveFilterButton" component 1`] = `
.c0 {
  color: #888888;
  margin-left: auto;
}

<div
  className="c0"
>
  RemoveFilterButton Test Text
</div>
`;
